decode_test.go

```diff
@@ -298,3 +298,26 @@ func tm(t *testing.T, layout, s string) time.Time {
 	}
 	return tm
 }
+
+func testEncode(t *testing.T) {
```

Comment:
I think a set of tests which compares the struct with the generated properties file content would be better. 

It should probably live in properties_test.go.

Reply:
Good point, will add.